src/main.c

```diff
@@ -308,12 +311,19 @@ static void focal_create_main_window(GApplication* app, FocalApp* fm)
 	gtk_header_bar_set_show_close_button(GTK_HEADER_BAR(fm->header), TRUE);
 	GtkWidget* nav = gtk_box_new(GTK_ORIENTATION_HORIZONTAL, 0);
 	gtk_style_context_add_class(gtk_widget_get_style_context(nav), "linked");
+
 	GtkWidget *prev = gtk_button_new(), *next = gtk_button_new();
+	char current_week_label[16];
+	sprintf(current_week_label, "Week %d", fm->todays_week);
```

Comment:
It looks to me that filling out the label of the button is the only reason the current date needs to be exposed outside of the `WeekView`. What if a symbolic icon is used instead, like the previous and next buttons? I think this would also be more consistent with the typical modern gtk app style. Maybe `find-location-symbolic` would be appropriate?

Reply:
Several other calendars seem to be simply labeling that button "today"

Reply:
Please see the updated PR: I for now used a textual button with unicode "▼" as text, which IMO fits better than an icon that in other contexts has a different meaning, and because it resembles the look of the other two navigation buttons.